README.md

```diff
@@ -22,13 +22,14 @@ This list contains tools, books, articles, blogs, courses and everything related
 - [Polish](https://github.com/mbiesiad/awesome-translations/tree/pl_PL)
 - [Spanish](https://web.archive.org/web/20220511155347/https://github.com/JoseDeFreitas/awesome-translations/tree/es_ES)
 - [Italian](https://github.com/niedev/awesome-translations/tree/it_IT)
+- [French](https://github.com/alexture/awesome-translations/blob/fr_FR/README.md)
```

Comment:
Awesome work! Looks great ❤ But here will be better to unified format like without `/blob`, ie. `https://github.com/alexture/awesome-translations/tree/fr_FR` I think